run.go

```diff
@@ -75,7 +75,11 @@ func Run(o Options) (err error) {
 
 	// Init window
 	var w *astilectron.Window
-	if w, err = a.NewWindow(filepath.Join(a.Paths().BaseDirectory(), resourcesPath, "app", o.Homepage), o.WindowOptions); err != nil {
+	var url = o.Homepage
+	if !strings.HasPrefix(o.Homepage, "http") && !strings.HasPrefix(o.Homepage, string(filepath.Separator)) {
```

Comment:
I've thought about it and I'd rather you use the following condition:

```go
if strings.Index(o.Homepage, "://") == -1 && !strings.HasPrefix(o.Homepage, string(filepath.Separator)) {
```

That way one can use a file named `http.html` for instance and any kind of protocol.